tests/BUILD

```diff
@@ -29,7 +29,7 @@ haskell_toolchain(
         "-DTESTS_TOOLCHAIN_COMPILER_FLAGS",
         "-XNoOverloadedStrings",  # this is the default, so it does not harm other tests
     ],
-    haddock_flags = ["-U"],
+    haddock_flags = ["-U", "-w"],
```

Comment:
I think disabling haddock warnings in general is the wrong move, because we won’t see warnings that do not come from generated files.

The problem here is that the interface files of the (compiled?) protobuf modules are not passed to the haddock process when it’s asked to build the docs. This happens here:

https://github.com/tweag/rules_haskell/blob/b24924dad34912326cb50d52afde685f93e2a776/haskell/haddock.bzl#L71-L75

You can see in the warnings how the `person.proto` has an `Address` field coming from the `address.proto`, and the Haddock process can’t find `Address` for `Proto.Tests.HaskellProtoLibrary.Person`. We somehow need to pass the interface files of protobuf modules to the haddock aspect. I wonder whether the protobuf modules generate any interfaces at all right now.